packages/bitcore-node/src/modules/firocoin/models/evmData.ts

```diff
@@ -0,0 +1,34 @@
+import { ObjectID } from 'mongodb';
+import { BaseModel } from '../../../models/base';
+import { StorageService } from '../../../services/storage';
+
+export interface IEvmData {
+  _id?: ObjectID;
+  chain: string;
+  network: string;
+  txid: string;
+  version: string;
+  fvmGasLimit: string;
+  fvmGasPrice: number;
+  callData: string;
+  contract: string;
+  op: string;
+}
+
+export class EvmDataModel extends BaseModel<IEvmData> {
+  constructor(storage?: StorageService) {
+    super('evmdatas', storage);
```

Comment:
data could not be pluralized like this `evmdata` is enough

Reply:
ok kub